### PR TITLE
source-bing-ads: migrate to per-stream state

### PR DESCRIPTION
**Pull request by alafanechere**
## What
We want source-bing-ads to use the latest CDK version.

## How
* Fix `accountId` key types in states (cast `int` to `str`)

### CODE REVIEW

File: airbyte-integrations/connectors/source-bing-ads/integration_tests/state.json [hunk 1]
```diff
@@ -1,22 +1,30 @@
-{
-  "keyword_performance_report_hourly": {
-    "180278106": {
-      "TimePeriod": 1627820152
+[
+  {
+    "type": "STREAM",
+    "stream": {
+      "stream_state": { "180278106": { "TimePeriod": 1627820152 } },
```
Comment (brianjlai) [inline airbyte-integrations/connectors/source-bing-ads/integration_tests/state.json:5]: tiny nit: can we indent these a to improve readability per level
```
"stream_state": {
  "180278106": {
    "TimePeriod": 1627820152
  }
}
```

**Review by brianjlai — APPROVED** [review]
Looks good to me! Nice find on the need to fix the way account was represented in the JSON. It's interesting that this was working in the first place since integers as a key are not considered valid JSON, so somewhere along the way we were translating the int to a str before persisting.

Also attaching a thread for more context on what was being observed during Pedro's OC.
https://airbytehq-team.slack.com/archives/C02TL38U5L7/p1663347369570809

**Comment by alafanechere** [comment]
/publish connector=connectors/source-bing-ads run-tests=false

**Pull request merged**